packages/flutter_tools/lib/src/base/logger.dart

```diff
@@ -1333,10 +1338,11 @@ class AnonymousSpinnerStatus extends Status {
 
   final List<String> _animation;
 
-  Timer? timer;
+  Timer? _timer;
+  @visibleForTesting
```

Comment:
Do we set `ticks` in tests anywhere? If not, should this be a getter for a private field?

Reply:
Fixed in [make ticks readonly, remove unnecessary protected annotations](https://github.com/flutter/flutter/pull/158492/commits/d81d598225749f37da3db436a6c6e34de00d430d)